cob_base_velocity_smoother/package.xml

```diff
@@ -8,13 +8,16 @@
   <url type="website">http://ros.org/wiki/cob_base_velocity_smoother</url>
   <!-- <url type="bugtracker"></url> -->
 
-  <maintainer email="frm@ipa.fhg.de">Florian Mirus</maintainer>
+  <maintainer email="bnm@ipa.fhg.de">Benjamin Maidel</maintainer>
   <author email="frm@ipa.fhg.de">Florian Mirus</author>
```

Comment:
suggestion from @ipa-fxm was to use the mail address of his GitHub Account: https://github.com/fmirus